target/linux/ath79/dts/ar9342_ubnt_nanobeam-ac.dts

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: GPL-2.0
```

Comment:
While I do like the idea of permissive licenses I have an internal conflict about using them in operating systems that are widely used with embedded devices.

Sadly a lot of manufacturers are not too keen on sharing information that allows third parties to build open firmware for their devices. Thus I feel like the GPL is a really important tool in focing vendors to provide that kind of information.
Even Ubiquiti who have not complied with GPL source code requests for a long time have finally given up and offer GPL source downloads for their firmware.

Since you can in fact do a lot of crazy stuff with device trees (especially regarding pin mappings which can be ridiculously hard to figure out) I do prefer GPL for all the code I do add to openwrt.

Please let me know if my above logic is flawed.

Reply:
@TobleMiner: this request is/was about DTS file only. File `.dts` gets compiled into `.dtb` and gets packaged into the device firmware file. It's not strictly part of the operating system.

Having `.dts` licensed as GPL wan't really force any vendor to share more of their code. They have hardware specs and can easily (with close to zero cost) write their own DTS. On the other hand having permissive licensed DTS file may result in vendors contributing easily (with no licensing fears) to it even if they use closed source bootloader using the same `.dts`.